scopes/scope/scope/scope.main.runtime.ts

```diff
@@ -397,7 +397,10 @@ needed-for: ${neededFor?.toString() || '<unknown>'}`);
   async getManifestsGraphRecursively(
     ids: string[],
     visited: string[] = [],
-    throwOnError = false
+    throwOnError = false,
+    opts: {
+      workspaceDir?: string
```

Comment:
Once you see something with the name workspace, inside the scope class, it should immediately pop up as red flag for you.
In general the scope is not aware to the workspace (workspace is aware to the scope).
It almost 100% means that the arg name should be replaced with a name that not represent it's value (like in this case), but its essence - pmConfigDir